File: client/src/Models/affixModel.ts
export interface Root {
  region: string
  title: string
  leaderboard_url: string
  affix_details: AffixModel[]
}

export interface AffixModel {
  icon_url: string
  id: number
  name: string
  description: string
  icon: string
  wowhead_url: string
}

File: server/models/affixModel.go
package models

type Affix struct {
	Region         string       `json:"region"`
	Title          string       `json:"title"`
	LeaderboardURL string       `json:"leaderboard_url"`
	AffixDetails   AffixDetails `json:"affix_details"`
}

type AffixDetails struct {
	ID          int    `json:"id"`
	Name        string `json:"name"`
	Description string `json:"description"`
	Icon        string `json:"icon"`
	WowheadURL  string `json:"wowhead_url"`
}
